modules/nodes-vsphere-machine-set-scaling-static-ip.adoc

```diff
@@ -0,0 +1,158 @@
+<2> Lists a nameserver. You must specify a nameserver for nodes that receive static IP address, because the Dynamic Host Configuration Protocol (DHCP) network configuration does not support static IP addresses. 
+
+. Scale the machine set by entering the following commands in your `oc` CLI:
++
+[source, terminal]
+----
+$ oc scale --replicas=2 machineset <machineset> -n openshift-machine-api
+----
++
+Or:
++
+[source, terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
++
+After each machine is scaled up, the machine controller creates an `IPAddresssClaim` resource.
+
+. Optional: Check that the `IPAddressClaim` resource exists in the `openshift-machine-api` namespace by entering the following command:
++ 
+[source, terminal]
+----
+$ oc get ipaddressclaims.ipam.cluster.x-k8s.io -n openshift-machine-api
+----
++
+.Example `oc` CLI output that lists two IP pools listed in the `openshift-machine-api` namespace
+[source, terminal]
+----
+NAME                                         POOL NAME        POOL KIND
+cluster-dev-9n5wg-worker-0-m7529-claim-0-0   static-ci-pool   IPPool
+cluster-dev-9n5wg-worker-0-wdqkt-claim-0-0   static-ci-pool   IPPool
+----
+
+. Create an `IPAddress` resource by entering the following command:
++
+[source, terminal]
+----
+$ oc create -f ipaddress.yaml
+----
++
+The following example shows an `IPAddress` resource with defined network configuration information and one defined static IP address:
++
+[source,yaml]
+----
+apiVersion: ipam.cluster.x-k8s.io/v1alpha1
+kind: IPAddress
+metadata:
+  name: cluster-dev-9n5wg-worker-0-m7529-ipaddress-0-0
+  namespace: openshift-machine-api
+spec:
+  address: 192.168.204.129
+  claimRef: <1>
+    name: cluster-dev-9n5wg-worker-0-m7529-claim-0-0
+  gateway: 192.168.204.1
+  poolRef: <2>
+    apiGroup: ipamcontroller.example.io
+    kind: IPPool
+    name: static-ci-pool
+  prefix: 23
+----
+<1> The name of the target `IPAddressClaim` resource. 
+<2> Details information about the static IP address or addresses from your nodes. 
++
+[NOTE]
+====
+By default, the external controller automatically scans any resources in the machine set for recognizable address pool types. When the external controller finds `kind: IPPool` defined in the `IPAddress` resource, the controller binds any static IP addresses to the `IPAddressClaim` resource.
+====
+
+. Update the `IPAddressClaim` status with a reference to the `IPAddress` resource:
++
+[source, terminal]
+----
+$ oc --type=merge patch IPAddressClaim cluster-dev-9n5wg-worker-0-m7529-claim-0-0 -p='{"status":{"addressRef": {"name": "cluster-dev-9n5wg-worker-0-m7529-ipaddress-0-0}}}' -n openshift-machine-api --subresource=status
```

Comment:
We missed a `"` here
`"cluster-dev-9n5wg-worker-0-m7529-ipaddress-0-0` 
shoud change to 
`"cluster-dev-9n5wg-worker-0-m7529-ipaddress-0-0"`

Reply:
Others lgtm